core/app.ts

```diff
@@ -105,6 +126,7 @@ function createApp (ssrContext, config): { app: Vue, router: VueRouter, store: S
   store.state.storeView = storeView
   // store.state.shipping.methods = shippingMethods
 
+  Vue.use(VueRouter)
```

Comment:
not sure if ssr will work as intended after moving this to createApp, have you tested?

Reply:
This seems to work as far as I can tell, the reason I moved this is because I think I need to call `use` after `beforeEach`. Not sure if this is the best place to put it though.

Reply:
I moved the logic to the user module so I reverted the `app.ts` file back to its original state.